From 5a2d0a794b4945f460902afda31427c2f44df2ce Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Wed, 3 Apr 2019 09:34:42 +0000
Subject: [PATCH] OS-7678 git pbchk copyright check isn't Joyent specific
 Reviewed by: Cody Peter Mello <melloc@writev.io> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/tools/onbld/Checks/Copyright.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/usr/src/tools/onbld/Checks/Copyright.py b/usr/src/tools/onbld/Checks/Copyright.py
index 8071b7f435..7727265622 100644
--- a/usr/src/tools/onbld/Checks/Copyright.py
+++ b/usr/src/tools/onbld/Checks/Copyright.py
@@ -25,6 +25,7 @@
 #
 
 # Copyright 2008, 2010, Richard Lowe
+# Copyright 2019, Joyent, Inc.
 
 # Make sure there is a copyright claim for the current year.
 
@@ -37,7 +38,7 @@ def is_copyright(line):
 	return re.search(r'Copyright (?!\[yyyy\])', line)
 
 def is_current_copyright(line):
-	return re.search(r'Copyright.*\b%s\b' % time.strftime('%Y'), line)
+	return re.search(r'Copyright %s, Joyent, Inc.$' % time.strftime('%Y'), line)
 
 def copyright(fh, filename=None, output=sys.stderr):
 	ret = rights = goodrights = 0
@@ -56,8 +57,7 @@ def copyright(fh, filename=None, output=sys.stderr):
 		err(output, "no copyright message found", filename)
 		ret = 1
 	elif goodrights == 0:
-		err(output, "no copyright claim for current year found",
-		    filename)
+		err(output, "no correct Joyent copyright found", filename)
 		ret = 1
 
 	return ret
-- 
2.21.0

